### PR TITLE
CHAIN-144: fixes to order book widget

### PR DESCRIPTION
**Pull request by bholzman**

### CODE REVIEW

File: mocker/src/main/kotlin/co/chainring/Main.kt [hunk 1]
```diff
@@ -26,15 +26,15 @@ fun main() {
...
+                "USDC" to 100.toFundamentalUnits(6),
+                "DAI" to 50.toFundamentalUnits(18)
             )
         )
     }
     takers.forEach {
         it.start(listOf(usdcDai))
     }
-    Thread.sleep(60000)
+    Thread.sleep(600000)
```
Comment (bflood366) [inline mocker/src/main/kotlin/co/chainring/Main.kt:37]: maybe default it to 600000 and also allow it to be passed in as a command line arg this way if we want to run this for a longer (or shorter) period we don't need to change the code.

**Review by bflood366 — APPROVED** [review]
looks great

File: mocker/src/main/kotlin/co/chainring/Main.kt (hunk 1 above)
Comment (bholzman) [reply mocker/src/main/kotlin/co/chainring/Main.kt:37]: I find myself making frequent edits here to try different scenarios, not sure command-line args would be the best way to parameterize -- thinking we need more like a yaml file to config. Will add a ticket but leave this as is.

Comment (antonyshchenko) [reply mocker/src/main/kotlin/co/chainring/Main.kt:37]: I would rather have an option to run this indefinitely until I kill the process myself.

**Pull request merged**